workflow/modules/3-align.smk

```diff
@@ -1,5 +1,10 @@
 import os
 
+# Set STAR parameter based on running on Azure Batch vs locally
+if "AZ_BATCH_IS_CURRENT_NODE_MASTER" in os.environ:
+    star_load_mode = "NoSharedMemory"
```

Comment:
Interesting... If we don't use shared memory the risk is that with many processes spawned it will use a lot more RAM than necessary